src/reducers/SupplierCreditReducer.js

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
pretty standard reducer - just mapping the itembatches to JS objects rather than using the realm objects is a bit different than normal.